packages/events/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@esri/hub-events",
+  "version": "1.3.0",
+  "description": "Module to interact with ArcGIS Hub Events in Node.js and modern browsers.",
+  "main": "dist/node/index.js",
+  "browser": "dist/umd/events.umd.js",
+  "module": "dist/esm/index.js",
+  "js:next": "dist/esm/index.js",
+  "types": "dist/esm/index.d.ts",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@esri/arcgis-rest-auth": "^1.11.0",
+    "@esri/arcgis-rest-common-types": "^1.11.0",
+    "@esri/arcgis-rest-feature-service": "^1.11.0",
+    "@esri/arcgis-rest-items": "^1.11.0",
+    "@esri/arcgis-rest-request": "^1.11.0",
+    "@esri/arcgis-rest-users": "^1.11.0",
+    "tslib": "^1.8.0"
+  },
+  "files": [
+    "dist/**"
+  ],
+  "scripts": {
+    "prepare": "npm run build",
+    "build": "npm run build:node && npm run build:umd && npm run build:esm",
+    "build:esm": "tsc --module es2015 --outDir ./dist/esm --declaration",
+    "build:umd": "rollup -c ../../umd-base-profile.js && rollup -c ../../umd-production-profile.js",
+    "build:node": "tsc --module commonjs --outDir ./dist/node",
+    "dev:esm": "tsc -w --module es2015 --outDir ./dist/esm --declaration",
+    "dev:umd": "onchange 'src/**/*.ts' -- npm run rollup:dev",
```

Comment:
I forgot to PR https://github.com/Esri/hub.js/pull/86 before you started on this, so you'll want to make [these changes](https://github.com/Esri/hub.js/pull/86/files#diff-75f00b8a9192522e46174597f8d2ffdb) here too. Sorry